app/components/admin/schools/cohorts/other_info.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Admin
+  module Schools
+    module Cohorts
+      class OtherInfo < BaseComponent
+        def initialize(cohort:, school_cohort:)
+          @cohort = cohort
+          @school_cohort = school_cohort
+        end
+
+      private
+
+        attr_reader :school_cohort, :cohort
+
+        MESSAGES = {
+          design_your_own: "designing own induction course",
+          school_funded_fip: "school funded full induction programme",
+          no_early_career_teachers: "no ECTs this year",
+        }.freeze
+
+        def message
+          return "No programme" if school_cohort.nil?
+
+          ["Not using service", MESSAGES[school_cohort.induction_programme_choice.to_sym]].compact.join(" - ")
```

Comment:
`.compact.join(" - ")` is nice and all, but would 
```
"Not using service - #{MESSAGES[school_cohort.induction_programme_choice.to_sym]}"
```
be a bit easier to read?

Reply:
Also, dies it deserve a component unit test?

Reply:
There's small difference in behaviour though - in case `MESSAGES[key]` returns nil, `.compact.join` will return `"Not using service"`, while string interpolation will yield `Not using service - ". Probably not a big issue though, but the idea is for that component to handle all sort of future types as well.

As per component test - probably should! Will add :)

Reply:
Hmmm, good point. I wonder if there is some way we could guard against merging code which adds a new option without adding a message here.

Oh well, that might be reason enough to keep the `compact.join` setup.